README.md

```diff
@@ -287,13 +287,13 @@ The method form takes parameters
 `auth(username, password, sendImmediately, bearer)`.
 
 `sendImmediately` defaults to `true`, which causes a basic or bearer
-authentication header to be sent.  If `sendImmediately` is `false`, then
```

Comment:
No problem -- it was requested that my new documentation remove double-spaces for consistency, so I wanted to fix other occurrences.